.jshintrc

```diff
@@ -10,8 +10,20 @@
   "strict": true,
   "undef": true,
   "unused": true,
+  "expr": true,
   "globals": {
     "angular": false,
-    "prettyPrintOne": false
+    "prettyPrintOne": false,
+    // MOCHA
```

Comment:
Seems to be right : https://github.com/gruntjs/grunt-contrib-jshint/blob/master/.jshintrc